### DISCUSSION TITLE
New release on curseforge/wowinterface?

### DISCUSSION THREAD

**Discussion by LunNova** [brittyazel/Neuron#484] [General]
Can we get a new release? @ylixir's fixes for classic dual spec would be great to have in a published version

**Comment by LunNova** [comment]
Oh. Didn't realize the main branch is currently an alpha.

It's working better than the previous branch did for classic so you can take that as good feedback :)

**Reply by ylixir** [reply]
I've been meaning to set up some kind of nightly or continuous build on github so that there would be an up to date version available for easier testing. It's been a little low priority, but maybe i should visit that soon

**Reply by LunNova** [reply]
That sounds good!

Might be worth adding something to the README.md in the main branch noting it's an alpha, currently it's not so obvious if you show up on github and I only worked that out after reading some discussions in the issues.

On the plus side it works well enough it doesn't feel like an alpha so maybe it doesn't need marked as alpha? :thinking:
